### PR TITLE
Add Ice.Exception values to output

### PR DESCRIPTION
**Pull request by kakty3**

### CODE REVIEW

File: src/yamlStream.ts [hunk 1]
```diff
@@ -47,27 +47,40 @@ const levelName = (level: number): string | undefined => {
...
   }
 
-  return stack.replace(basePath, '') + '\n';
+  const stackLines = stack.split('\n');
+  const errorContext = {
+    context: omit(err, ['stack', 'message', 'ice_name', 'ice_cause']),
+  };
+  const stackTrace = stackLines.slice(1).map(
+    line => line.replace(basePath, '')
+  );
```
Comment (3g0r) [inline src/yamlStream.ts]: may be better `const stackLines = stack.replace(basePath, '').split('\n');`

Comment (kakty3) [reply src/yamlStream.ts]: It is possible to have `basePath` in exception's message. In your case message will be modified as well, but it is unwanted.

Comment (3g0r) [reply src/yamlStream.ts]: Ok, then:
```
const stackLines = stack.replace(new RegExp(`(\\()(${escapeRegExp(basePath)})`), '$1').split('\n');
```

Comment (kakty3) [reply src/yamlStream.ts]: Message could contain any possible string so we should not modify message anyway

**Pull request merged**